models/user.py

```diff
@@ -1,9 +1,9 @@
-from pydantic import BaseModel, EmailStr
+from beanie import Document
+from beanie import init_beanie
 
-class User(BaseModel):
+from controllers.auth import connect_to_db
```

Comment:
remove this

Reply:
from beanie import init_beanie
from controllers.auth import connect_to_db
remove....

Reply:
from beanie import Document
from pydantic import EmailStr

this is the imports i have, i dont think i need to import init_beanie...